src/typescript/azure-openapi-validator/rules/LintDescriptionProse.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const LintDescriptionProse: string = "LintDescriptionProse";
+
+import * as jp from 'jsonpath';
+import { spawnSync } from 'child_process';
+
+var noLinterWarning = false;
+
+rules.push({
+  id: "D4100",
+  name: LintDescriptionProse,
+  severity: "warning",
+  category: "DocViolation",
+  mergeState: MergeStates.composed,
+  openapiType: OpenApiTypes.doc,
+  appliesTo_JsonQuery: "$..description",
+  run: function* (doc, node, path) {
+    const msg: string = "`proselint` generated warnings: ";
```

Comment:
what type of things it proselint checking? do you have a link to extra documentation on it?

Reply:
The full `proselint` documentation is available. https://github.com/amperser/proselint/